app/views/sessions/new.html.erb

```diff
@@ -27,7 +27,7 @@
     <%= render :partial => "auth_providers" %>
     <div class="d-flex justify-content-center align-items-center">
       <div class="border-bottom border-1 flex-grow-1"></div>
-      <div class="text-secondary mx-3"><%= t ".or" %></div>
+      <div class="text-secondary mx-3 mb-2"><%= t ".or" %></div>
```

Comment:
Was there a reason for choosing the `mb-2` rather than `mb-3` that @gravitystorm suggested?